README.md

```diff
@@ -34,6 +34,7 @@ We publish blogs on Medium, so [follow us](https://medium.com/intel-analytics-so
 - [Building from Sources](https://github.com/intel/scikit-learn-intelex/blob/master/INSTALL.md)
 - [About Intel(R) oneAPI Data Analytics Library](https://github.com/oneapi-src/oneDAL)
 - [About Intel(R) daal4py](https://github.com/intel/scikit-learn-intelex/tree/master/daal4py)
+- [Jupyter examples](https://github.com/intel/scikit-learn-intelex/tree/master/examples/notebooks)
```

Comment:
Move to first

Reply:
not applied

Reply:
To be honest - they are not jupyter notebooks.  jupyter - just a way how to work with .ipynb.
So, better to say "Python notebook examples" or "Notebook examples"

Reply:
Thanks, fixed